SDL2_OpenGL_BoilerPlate/BoilerPlate/App.cpp

```diff
@@ -129,24 +143,23 @@ namespace Engine
 		double startTime = m_timer->GetElapsedTimeInSeconds();
 
 		// Update code goes here
-		//
+		game->player->Update(m_width, m_height, DESIRED_FRAME_TIME);
```

Comment:
Updates should be handled by the game class. Remember to use **deltatime**, you will need it for the framerate performance plot.